frontend/components/welcomeHeader.jsx

```diff
@@ -17,27 +17,58 @@ const TitleContainer = styled.div`
 `;
 
 const TimerContainer = styled.div`
-  align-self: flex-end;
+`;
+
+const InfoContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const TableContainer = styled.div`
+`;
+
+const SolvedContainer = styled.div`
 `;
 
 class WelcomeHeader extends React.Component {
-  render() {
-    const { gameState } = this.props;
+  constructor(props) {
+    super(props);
+    this.renderSolved = this.renderSolved.bind(this);
+  }
 
-    const { start_time } = gameState || {};
+  renderSolved(gameState) {
```

Comment:
usually when a function is prefixed with render, it will render something, but this just returns the count, maybe something like numSolved is better